helm/templates/networkPolicies/networkPolicy.yaml

```
@@ -0,0 +1,21 @@
+{{- if .Values.networkPolicies.enabled }}
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  name: {{ .Release.Name }}
```

```suggestion
  name: {{ .Release.Name }}-allow-route-ingress
```
You'll want to use descriptive names for your network policies